test/test_client.py

```diff
@@ -8,6 +8,64 @@
 from radiant_mlhub.exceptions import EntityDoesNotExist, MLHubException, AuthenticationError
 
 
+
+class TestCustomUrl:
+
+    def test_custom_url_list_datasets(self, monkeypatch, requests_mock):
+        # Set up custom URL
+        custom_root_url = "https://staging.api.radiant.earth"
+        monkeypatch.setenv('MLHUB_ROOT_URL', custom_root_url)
+
+        # Mock this using requests-mock
+        url = 'https://staging.api.radiant.earth/datasets?key=test_key'
+        requests_mock.get(url, status_code=200, json=[])
+
+        # Making request to API
+        radiant_mlhub.client.list_datasets()
+
+        # Get request history and check that request was made to custom URL
+        history = requests_mock.request_history
+        assert len(history) == 1
+        assert history[0].url == "https://staging.api.radiant.earth/datasets?key=test_key"
+
+
+    def test_custom_url_get_collection(self, monkeypatch, requests_mock):
+        # Set up custom URL
+        custom_root_url = "https://staging.api.radiant.earth"
+        monkeypatch.setenv('MLHUB_ROOT_URL', custom_root_url)
+
+        # Mock this using requests-mock
+        url = 'https://staging.api.radiant.earth/datasets?key=test_key'
+        requests_mock.get(url, status_code=200, json=[])
+
+        # Making request to API
+        radiant_mlhub.client.get_collection("collection_id")
+
+        # Get request history and check that request was made to custom URL
+        history = requests_mock.request_history
+        assert len(history) == 1
+        assert history[0].url == "https://staging.api.radiant.earth/datasets?key=test_key"
```

Comment:
```suggestion
        assert history[0].url == "https://staging.api.radiant.earth/collections/collection_id?key=test_key"
```

This will also need to be updated to match the URL used by `radiant_mlhub.client.get_collection`.